changelog/_template.jinja2

```diff
@@ -0,0 +1,18 @@
+{% for section in sections %}{% set underline = "-" %}{% if section %}{{section}}{{ underline * section|length }}{% set underline = "~" %}
+
```

Comment:
This created a ton of empty lines for me, is that on purpose? @tmbo 

```
[1.5.1] - 2019-11-28
^^^^^^^^^^^^^^^^^^^^


Improvements
------------

- `#3684 <https://github.com/rasahq/rasa/issues/3684>`_: ``rasa interactive`` will skip the story visualization of training stories in case
  there are more than 200 stories. Stories created duringinteractive learning will be
  visualized as before.



Bugfixes
--------

- `#3684 <https://github.com/rasahq/rasa/issues/3684>`_: ``rasa interactive`` will not load training stories in case the visualization is
  skipped.



Improved Documentation
----------------------

- `#4844 <https://github.com/rasahq/rasa/issues/4844>`_: Added ``towncrier`` to automatically collect changelog entries.



Miscellaneous internal changes
------------------------------

 - #4844
```

Reply:
not on purpose, this is mostly the default template - I removed most of them.

Reply:
There are still some too many (compared to our old one):

```
[1.5.1] - 2019-11-28
^^^^^^^^^^^^^^^^^^^^
2 instead of 1

Bugfixes
--------
1 instead of 0
- `#4802 <https://github.com/rasahq/rasa/issues/4802>`_: Fixed unnecessary ``kwargs`` unpacking in ``rasa.test.test_core`` call in ``rasa.test.test`` function.

Improved Documentation
----------------------

- `#4844 <https://github.com/rasahq/rasa/issues/4844>`_: Added ``towncrier`` to automatically collect changelog entries.

Miscellaneous internal changes
------------------------------

 - #4844
```

Reply:
You are right :+1: removed them (there is also one additional newline at the end, but couldn't find it right away - don't think it makes a difference, so I guess we'll just keep it)